zulipterminal/config/tutorial.py

```diff
@@ -0,0 +1,24 @@
+TUTORIAL = """
+
+Basic workflows
+
+A few basic workflows are useful when getting oriented.
+
+Moving around
+
+- You can use arrow keys or `j`/`k`/`l`/`m` keys to move around and
+ `esc` to go back or move out of narrow.
+
+For reading
+
+- Use the left sidebar (or the `n` key) to review messages thread-by-thread.
+- Use the down arrow key (`↓`/`j`) to move the cursor to the next message.
+- Use the End key to mark all messages in the current view as read.
+
+For writing
+
+- To reply to a message, click on it, or use `r` or `Enter`.
+- You can type your message in the bottom big box and send it using `ctrl d` or `alt enter`.
+
+\n\n
+"""
```

Comment:
I think we should mention at the end that pressing `?` would open up the help menu for more.